pre_install_report/README.md

```diff
@@ -72,3 +73,19 @@ The tool generates the pre-install check report,`viya_pre_install_report_<timest
 ## Modify CPU, Memory, and Version Settings
 
 You can modify the <tool-download-dir>/viya4-ark/pre_install_report/viya_check_limit.properties file to alter the minimum and aggregate settings for CPU and memory on nodes. For more information, see the details in the file.
+
+## Known Issues
+
+The following issue may impact the performance and expected results of this tool.
+- All Nodes in a cluster must be in the READY state before running the tool.
+    - If all the Nodes are not in the READY state, the tool takes longer to run. Wait for it to complete.
+    - Also, the tool may not be able to clean up the pods and replicaset created in the specified namespace as shown in the example output below. If that happens, the pods and replicaset must be manually deleted.
```

Comment:
Can we offer instructions on how to manually delete the pods and replicaset?

Reply:
Yes I will add the commands to delete the resources that were not cleaned up

Reply:
I pushed my changes now.